docs/core/whats-new/dotnet-core-3-0.md

```diff
@@ -0,0 +1,496 @@
+
+```csharp
+async IAsyncEnumerable<int> GetBigResultsAsync()
+{
+    await foreach (var result in GetResultsAsync())
+    {
+        if (result > 20) yield return result; 
+    }
+}
+```
+
+In addition to being able to `await foreach`, you can also create async iterators, for example, an iterator that returns an `IAsyncEnumerable/IAsyncEnumerator` that you can both `await` and `yield` in. For objects that need to be disposed, you can use `IAsyncDisposable`, which various BCL types implement, such as `Stream` and `Timer`.
+
+> [!NOTE]
+> Only [C# 8.0](https://blogs.msdn.microsoft.com/dotnet/2018/11/12/building-c-8-0/) supports `await foreach` syntax.
+
+## SequenceReader
+
+In .NET Core 3.0, `System.Buffers.SequenceReader` has been added which can be used as a reader for `ReadOnlySequence<T>`. This allows easy, high performance, low allocation parsing of `System.IO.Pipelines` data that can cross multiple backing buffers. 
+
+The following example breaks an input `Sequence` into valid `CR/LF` delimited lines:
+
+```csharp
+private static ReadOnlySpan<byte> CRLF => new byte[] { (byte)'\r', (byte)'\n' };
+
+public static void ReadLines(ReadOnlySequence<byte> sequence)
+{
+    SequenceReader<byte> reader = new SequenceReader<byte>(sequence);
+
+    while (!reader.End)
+    {
+        if (!reader.TryReadToAny(out ReadOnlySpan<byte> line, CRLF, advancePastDelimiter:false))
+        {
+            // Couldn't find another delimiter
+            // ...
+        }
+
+        if (!reader.IsNext(CRLF, advancePast: true))
+        {
+            // Not a good CR/LF pair
+            // ...
+        }
+
+        // line is valid, process
+        ProcessLine(line);
+    }
+}
+```
+
+## TLS 1.3 & OpenSSL 1.1.1 on Linux
+
+.NET Core will now take advantage of [TLS 1.3 support in OpenSSL 1.1.1](https://www.openssl.org/blog/blog/2018/09/11/release111/), when it is available in a given environment. There are multiple benefits of TLS 1.3, per the [OpenSSL team](https://www.openssl.org/blog/blog/2018/09/11/release111/):
+
+* Improved connection times due to a reduction in the number of round trips required between the client and server.
+
+* Improved security due to the removal of various obsolete and insecure cryptographic algorithms and encryption of more of the connection handshake.
+
+.NET Core 3.0 Preview 1 is capable of utilizing **OpenSSL 1.1.1**, **OpenSSL 1.1.0**, or **OpenSSL 1.0.2** (whatever the best version found is, on a Linux system).  When **OpenSSL 1.1.1** is available the SslStream and HttpClient types will use **TLS 1.3** when using `SslProtocols.None` (system default protocols), assuming both the client and server support **TLS 1.3**.
+
+The following sample demonstrates .NET Core 3.0 Preview 1 on Ubuntu 18.10 connecting to https://www.cloudflare.com:
```

Comment:
Markdig will complain that https://www.cloudflare.com isn't formatted as a link.

Reply:
Are you sure? Or is it just when it detects www without http ? 

https://github.com/lunet-io/markdig/blob/master/src/Markdig.Tests/Specs/AutoLinks.md

Reply:
Well, i'm not certain. :shipit:

Reply:
It's a problem for Loc. Use `<https://www.cloudflare.com>`